scripts/openshiftci-presubmit-all-tests.sh

```diff
@@ -2,63 +2,31 @@
 
 # fail if some commands fails
 set -e
+# Hide command that deals with secrets
+set +x
+if [[ -f $ODO_RABBITMQ_AMQP_URL ]]; then
+    export AMQP_URI=$(cat $ODO_RABBITMQ_AMQP_URL)
+fi
+if [[ -z $AMQP_URI ]]; then
+    echo "Please set AMQP_URI env with amqp uri or provide path of file containing it as ODO_RABBITMQ_AMQP_URL env"
+    exit 1
+fi
+export JOB_NAME="odo-pr-build"
+export REPO_URL="https://github.com/openshift/odo"
+export TARGET="$(jq .refs.pulls[0].number <<< $(echo $JOB_SPEC))"
+export CI_FIREWALL_VERSION="valpha"
```

Comment:
Would it make more sense to initialize these variables at the beginning of the file? Asking because that's how I have usually seen it done in shell scripts. 

Also, just for the sake of my understanding, do we really need them after the execution of this script is over? Asking because they're `export`ed.

Reply:
Export exports env to the current shell (which is basically the one running this script) and all child processes. 

If I don't export it, I will have to set it with every process that needs the env like 
```
ENV1=someval command1;
ENV1=someval command2;
```